packages/bot-skeleton/src/scratch/hooks/block_svg.js

```diff
@@ -187,6 +187,16 @@ Blockly.BlockSvg.prototype.setErrorHighlighted = function(
     this.error_message = error_message;
 };
 
+// Highlight the block that is being executed
+Blockly.BlockSvg.prototype.highlightExecutedBlock = function() {
+    const highlight_block_class = 'block--execution-highlighted';
```

Comment:
It's better if you use [BEM structure ](http://getbem.com/naming/)for class names

Reply:
I've followed the same pattern as the error highlight class on line 177:
`const highlight_class = 'block--error-highlighted'`.
Also asked @nijil-binary and he said it follows BEM. 😅